ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -175,7 +158,7 @@ function RequestCVEsTable({
                                 );
 
                                 return (
-                                    <Tbody key={cve}>
+                                    <Tbody key={cve} isExpanded={isExpanded}>
```

Comment:
The `isExpanded={isExpanded}` prop for both `Tbody` and second `Tr` sounds like a nice learning (or annoyance, depending on your viewpoint) for team meeting next Friday.